### PR TITLE
Update server configuration docs

### PR DESCRIPTION
**Pull request by 12AwsomeMan34**
Updates the server configuration docs.

Current changes:
- /sponge conf -> /sponge config
- Getting the value from a config
- saving and reloading world configs
- Update config example to build 1000

WIP list:
- [x] Update 'Global Properties of Sponge'
- [x] Update commands page to match /sponge config

and any other references to server configuration that is outdated.

### CODE REVIEW

**Comment by Tzky** [comment]
I suggest using a code block if you need a whole line of code instead of inline codeblocks:
> .. code-block:: none

creates such a codeblock. you can choose different code highlighting options, for example ``none``, ``java`` and some others.

File: source/server/getting-started/configuration/sponge-conf.rst [hunk 1]
```diff
@@ -165,109 +176,107 @@ running SpongeForge (build 516) and Forge 1.8 (build 1487).
...
 **Minecraft Creatures**
-* Ambient (bat)                 boolean   true      Use activation range on bats
-* Aquatic (squid)               boolean   true      Use activation range on squid.
-* Creature <creature>           boolean   true      Use activation range on <creature>. Permitted
-                                                    values: chicken, cow, entityhorse, mushroomcow,
-                                                    ozelot, pig, rabbit, sheep, wolf.
-enabled                         boolean   true      Enable activation range on Minecraft entities.
+* Ambient (bat)                  boolean   true        Use activation range on bats
+* Aquatic (squid)                boolean   true        Use activation range on squid.
+* Creature <creature>            boolean   true        Use activation range on <creature>. Permitted
```
Comment (Tzky) [inline source/server/getting-started/configuration/sponge-conf.rst]: i'd remove the bulletpoints here

**Comment by Tzky** [comment]
One last note: 
Your forks master branch seems outdated. However we can rebase this PR before merging it, if you don't want to mess with your fork.

**Comment by Tzky** [comment]
Looks good to me, i haven't checked every value though ;)

File: source/server/getting-started/configuration/index.rst [hunk 2]
```diff
@@ -42,25 +42,69 @@ Dimension configuration files are used to affect a certain dimension or group of
...
 override dimension and global configs.
 
-It is possible to modify these configs through the in-game command ``/sponge conf``.
-
-The sponge conf command syntax looks like this:
-
-``/sponge conf <flag> <name> <key> <value>``
-
-The *flag* is the group you are targeting, such as global, dimension, or world.
+There are *flags* for specifying the target that you would wish to change. These flags are global, dimension, and world.
```
Comment (saladoc) [inline source/server/getting-started/configuration/index.rst]: Flags are explicitly related to the command. Moving the command down and leaving the flags here does not make any sense.

File: source/server/getting-started/configuration/index.rst [hunk 3]
```diff
@@ -42,25 +42,69 @@ Dimension configuration files are used to affect a certain dimension or group of
...
 
+Modifying the Config In-Game
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to modify these configs through the in-game command ``/sponge config``. The syntax for the config
+command looks like this:
+
+.. code-block:: java
+
+    /sponge config <flag> <name> <key> <value>
```
Comment (saladoc) [inline source/server/getting-started/configuration/index.rst]: the `name` parameter is not explained. I assume it's the world / dimension name? Then it's probably part of the flags anyway.

**Comment by Tzky** [comment]
Squash please, then we'll merge. :)

**Comment by Tzky** [comment]
closed via f2b9312

**Pull request closed without merging**